ingestion_tools/scripts/db_import.py

```diff
@@ -23,19 +23,40 @@ def cli():
     pass
 
 
+def db_import_options(func):
+    options = []
+    options.append(click.option("--import-annotations", is_flag=True, default=False))
+    options.append(click.option("--import-annotation-authors", is_flag=True, default=False))
+    options.append(click.option("--import-dataset-authors", is_flag=True, default=False))
+    options.append(click.option("--import-dataset-funding", is_flag=True, default=False))
+    options.append(click.option("--import-runs", is_flag=True, default=False))
+    options.append(click.option("--import-tiltseries", is_flag=True, default=False))
+    options.append(click.option("--import-tomograms", is_flag=True, default=False))
+    options.append(click.option("--import-tomogram-authors", is_flag=True, default=False))
+    options.append(click.option("--import-tomogram-voxel-spacing", is_flag=True, default=False))
+    options.append(click.option("--import-everything", is_flag=True, default=False))
+    options.append(
+        click.option(
+            "--anonymous",
+            is_flag=True,
+            required=True,
+            default=False,
+            type=bool,
+            help="Use anonymous access to S3",
+        ),
+    )
+    for option in options:
+        func = option(func)
+    return func
+
+
 @cli.command()
 @click.argument("s3_bucket", required=True, type=str)
 @click.argument("https_prefix", required=True, type=str)
 @click.argument("postgres_url", required=True, type=str)
-@click.option("--s3_prefix", required=True, default="", type=str)
-@click.option(
-    "--anonymous",
-    is_flag=True,
-    required=True,
-    default=False,
-    type=bool,
-    help="Use anonymous access to S3",
-)
+@click.option("--filter-dataset", type=str, default=None, multiple=True)
+@click.option("--s3-prefix", required=True, default="", type=str)
+@click.option("--endpoint-url", type=str, default=None)
 @click.option(
```

Comment:
What are your thoughts on adding `--debug` to the db_import_options and having it default to True for any time we run db_import in staging/prod?

Reply:
good call! added that.